### PR TITLE
#122 Add starters POM dependencies diagram

### PR DESCRIPTION
**Pull request by tzolov**
Add layered dependency diagram with brief explanation of the important dependency relationships

Resolves spring-cloud-stream-app-starters/app-starters-release#122

### CODE REVIEW

File: spring-cloud-stream-app-starters-docs/src/main/asciidoc/pom-dependencies.adoc [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+== App Starter POM Dependencies
+
+Following diagram highlights the important App Starter POM dependencies.
+
+image:images/starters-pom-dependencies.png[PomDependencies, scaledwidth="75%"]
+
+The dependencies are grouped in three categories:
+
+* *Core Spring Dependencies* - represent the core framework libraries such as `Spring Boot`, `Spring Integration`,
+`Spring Cloud`. The "Bill Of Materials" (BOM) patterns is actively used thought the stack to decouple the
```
Comment (artembilan) [inline spring-cloud-stream-app-starters-docs/src/main/asciidoc/pom-dependencies.adoc]: Typo: s/thought/throughout

**Review by artembilan — CHANGES_REQUESTED** [review]
Excellent! 👍

File: spring-cloud-stream-app-starters-docs/src/main/asciidoc/pom-dependencies.adoc (hunk 1 above)
Comment (olegz) [reply spring-cloud-stream-app-starters-docs/src/main/asciidoc/pom-dependencies.adoc]: @artembilan @tzolov I am ok to fix it during the merge. No need to submit an updated PR

**Review by artembilan — APPROVED** [review]
Sure, @olegz ! Ship it!

However right now there is just enough to press "Squash and merge" button here on GH 😄 .

@tzolov ,
Thank you!

**Pull request merged**